doc/source/whatsnew/v0.25.0.rst

```diff
@@ -159,29 +159,53 @@ cause a ``SparseSeries`` or ``SparseDataFrame`` to be returned, as before.
 Increased minimum versions for dependencies
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Due to dropping support for Python 2.7, a number of optional dependencies have updated minimum versions.
-Independently, some minimum supported versions of dependencies were updated (:issue:`23519`, :issue:`24942`).
+Due to dropping support for Python 2.7, a number of optional dependencies have updated minimum versions (issue:`25725`, :issue:`24942`, :issue:`25752`).
+Independently, some minimum supported versions of dependencies were updated (:issue:`23519`, :issue:`25554`).
 If installed, we now require:
 
-+-----------------+-----------------+----------+
-| Package         | Minimum Version | Required |
-+=================+=================+==========+
-| beautifulsoup4  | 4.4.1           |          |
-+-----------------+-----------------+----------+
-| openpyxl        | 2.4.0           |          |
-+-----------------+-----------------+----------+
-| pymysql         | 0.7.9           |          |
-+-----------------+-----------------+----------+
-| pytz            | 2015.4          |          |
-+-----------------+-----------------+----------+
-| sqlalchemy      | 1.1.4           |          |
-+-----------------+-----------------+----------+
-| xlsxwriter      | 0.7.7           |          |
-+-----------------+-----------------+----------+
-| xlwt            | 1.0.0           |          |
-+-----------------+-----------------+----------+
-| pytest (dev)    | 4.0.2           |          |
-+-----------------+-----------------+----------+
++-----------------+-----------------+-----------------------+
+| Package         | Minimum Version | Comment               |
++=================+=================+=======================+
+| numpy           | 1.13.3          | Required              |
++-----------------+-----------------+-----------------------+
+| pytz            | 2015.4          | Required              |
++-----------------+-----------------+-----------------------+
+| bottleneck      | 1.2.1           | Required if installed |
++-----------------+-----------------+-----------------------+
+| numexpr         | 2.6.2           | Required if installed |
++-----------------+-----------------+-----------------------+
+| pytest          | 4.0.2           | Development-only req. |
```

Comment:
this is a misleading comment for pytest. its required if you run tests

Require if installed is very misleading, remove it